### PR TITLE
Feature/cypress

### PR DESCRIPTION
**Pull request by Syed-Vtex**

### CODE REVIEW

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+name: '[QE] Cypress Runner'
+
+on:
+  workflow_dispatch:
+  schedule:
+    # 8:30 AM UTC = 4:30 AM EDT
+    - cron: '30 8 * * *'
```
Comment (thyarles) [inline .github/workflows/cypress.yml:7]: Are you sure to schedule it right now?

File: .github/workflows/cypress.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      - docs/**
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: false
+
+jobs:
+  cypress:
+    runs-on: ubuntu-latest
+    # To avoid waste Cypress tests,
```
Comment (thyarles) [inline .github/workflows/cypress.yml:28]: Please, add a `timeout-minutes: 30` to define the max time this job can consume.

File: .github/workflows/cypress.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        env:
+          VTEX_QE: ${{ secrets.VTEX_QE }}
+          NODE_NO_WARNINGS: 1
+
+      - name: Save results
+        uses: actions/upload-artifact@master
+        if: always()
+        with:
+          name: cy-runner-logs
+          path: cy-runner/logs
```
Comment (thyarles) [inline .github/workflows/cypress.yml:64]: Please, add `retention-days: 3` here.

File: cy-runner.yml [hunk 4]
```diff
@@ -0,0 +1,103 @@
+---
+base:
+  secrets:
+    enabled: true
+    name: VTEX_QE
+  vtex:
+    account: productusqa
+    id: 2001459804
```
Comment (thyarles) [inline cy-runner.yml:8]: If we don't need the ID, take it out.

File: cy-runner.yml [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    domain: myvtex.com
+    urlExternalSeller: https://productusqaseller.myvtex.com
```
Comment (thyarles) [inline cy-runner.yml:10]: If we don't need urtExternalSeller, take it out

File: .github/workflows/cypress.yml (hunk 1 above)
Comment (Syed-Vtex) [reply .github/workflows/cypress.yml:7]: yes
8 am utc is b2b
So, I used 8:30 am utc for checkout ui custom

File: cypress/integration/2.3-checkout_scenarios.spec.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    cy.addProduct(productName)
+    // Remove the product
+    cy.removeProduct('880300')
+    // Search the product
+    cy.searchProduct(productName)
+    // Add product to cart
+    cy.addProduct(productName)
+  })
+
+  it(`In ${prefix} Updating product quantity to 2`, updateRetry(3), () => {
```
Comment (thyarles) [inline cypress/integration/2.3-checkout_scenarios.spec.js:28]: It's really necessary update quantity again? I saw it on the previous test.

File: cypress/integration/2.3-checkout_scenarios.spec.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+      quantity: '2',
+    })
+  })
+
+  it(`In ${prefix} - Updating Shipping Information`, updateRetry(3), () => {
+    // Update Shipping Section
+    cy.updateShippingInformation(checkoutScenario)
+  })
+
+  it(`In ${prefix} Updating product quantity to 1`, updateRetry(2), () => {
```
Comment (thyarles) [inline cypress/integration/2.3-checkout_scenarios.spec.js:40]: It's really necessary update quantity again? I saw it on the previous test.

File: cypress/integration/2.4-custom_js_css.spec.js [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+  const { productName, postalCode, prefix } = testCustomJSAndCSS
+
+  it(`In ${prefix} - Adding Product to Cart`, updateRetry(3), () => {
+    // Search the product
+    cy.searchProduct(productName)
+    // Add product to cart
+    cy.addProduct(productName)
+  })
+
+  it(`In ${prefix} Updating product quantity to 2`, updateRetry(3), () => {
```
Comment (thyarles) [inline cypress/integration/2.4-custom_js_css.spec.js:18]: It's really necessary update quantity again? I saw it on the previous test.

**Review by thyarles — CHANGES_REQUESTED** [review]
Some suggestions.

File: cypress/integration/2.3-checkout_scenarios.spec.js (hunk 6 above)
Comment (Syed-Vtex) [reply cypress/integration/2.3-checkout_scenarios.spec.js:28]: yes it is a testcase where we check if quantity gets increased total gets increased/ if quantity gets decreased the total also gets decreased

File: cypress/integration/2.4-custom_js_css.spec.js (hunk 8 above)
Comment (Syed-Vtex) [reply cypress/integration/2.4-custom_js_css.spec.js:18]: I will remove it in next commit

File: cy-runner.yml [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+    enabled: false
+    channel: some-channel
+  cypress:
+    devMode: false
+    runHeaded: false
+    getCookies: true
+    maxJobs: 3
+    quiet: true
+    projectId: x6id56
+    video: true
```
Comment (thyarles) [inline cy-runner.yml:33]: Please, turn off the video.

File: cypress/integration/2.3-checkout_scenarios.spec.js (hunk 6 above)
Comment (thyarles) [reply cypress/integration/2.3-checkout_scenarios.spec.js:28]: But do it one time in one test isn't enough to assure it is working? You are doing it three times... and update/decrease quantity is a flaky test as we can see on Cypress Dashboard.

**Review by thyarles — APPROVED** [review]
LGTM

**Pull request merged**